ast/var_decl.c2

```diff
@@ -228,6 +229,14 @@ public fn void VarDecl.setLocal(VarDecl* d, bool has_local) {
     d.base.varDeclBits.has_local = has_local;
 }
 
+public fn void VarDecl.setInitCall(VarDecl* d, bool has_init_call) {
+    d.base.varDeclBits.has_init_call = has_init_call;
+}
+
+public fn bool VarDecl.hasInitCall(VarDecl* d) {
+    return d.base.varDeclBits.has_init_call;
+}
+
```

Comment:
Could you also add printing it in the VarDecl.print() function?

Reply:
Done